frontend/src/charts/Charts.types.ts

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+export interface GQLRepresentationMetadata {
+  id: string;
+  label: string;
+  kind: string;
+  description: GQLRepresentationDescription;
+}
+
+export interface GQLRepresentationDescription {
+  id: string;
+}
+export interface GQLRepresentation {
+  id: string;
+}
+export interface Chart extends GQLRepresentation {
```

Comment:
We have chosen this approach because the data you get from the subscription can be transformed. There is an example of such transformation in the validation websocket container state machine.